lib/src/widgets/slidable.dart

```diff
@@ -323,15 +323,15 @@ class SlidableData extends InheritedWidget {
       (oldWidget.direction != direction);
 }
 
-/// A controller that keep tracks of the active [SlidableState] and close
+/// Creates a controller that keep tracks of the active [SlidableState] and close
```

Comment:
Why change the comment of the class? the class itself does not create a controller, the constructor does.